Docs: TAG policy revision (#42121)
gravitational/teleport: docs/pages/access-controls/access-graph/aws-sync.mdx

---
title: Discover AWS Access Patterns with Teleport Policy
description: Describes how to import and visualize AWS accounts access patterns using Teleport Policy and Access Graph.
---

> Teleport Policy will streamline and centralize access management across your entire infrastructure. You can view access relationships in seconds, 

Reviewer: ```suggestion
Teleport Policy streamlines and centralizes access management across your entire infrastructure. You can view access relationships in seconds, 
```

<Notice type="warning">

> If you have a Teleport Enterprise cluster, you can disregard

Reviewer: ```suggestion
If you have a managed Teleport Enterprise cluster, you can disregard
```

> this step, as Teleport Enterprise already operates a properly configured 

Reviewer: ```suggestion
this step, as managed Teleport Enterprise already operates a properly configured 
```
Self-hosted Teleport Enterprise would still require you to deploy Access Graph
